addons/mass_mailing/data/mail_data.xml

```diff
@@ -17,4 +17,273 @@
             </body>
         </html>
     </template>
+
+    <!-- DIGEST TIPS -->
+
+    <record id="digest_tip_mass_mailing_0" model="digest.tip">
```

Comment:
Meh. This will add tips in digest, we have a task for that (cleaning tips and digest). Are you sure those should be mixed with standard tips ? Could you check with LNA ?

Reply:
Yes, it should be mixed with standard tips (but with high sequence)